elasticsearch/html/assets/js/example.js

```diff
@@ -0,0 +1,36 @@
+$( document ).ready(function() {
+  var client = new elasticsearch.Client({
+    host: 'http://elasticsearch.developers.loc',
```

Comment:
maybe we can read this from somewhere?

Reply:
You mean to put it in a configuration file?